spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -9,14 +9,6 @@
       create(:service_provider, :active, :in_person_proofing_enabled)
     end
 
-    scenario 'home page' do
-      sign_in_and_2fa_user
-
-      visit idv_path
-
-      expect_page_to_have_no_accessibility_violations(page)
-    end
-
     scenario 'how to verify page' do
```

Comment:
Should we consolidate this one in as well?

Reply:
It gets kind of difficult since it's not in the default happy path. Enabling in-person-proofing adds a screen in the middle. We could probably modify the existing step helpers to accommodate it though (especially since it will be enabled more widely soon).

I'm going to merge this for now though.